pride/src/main/java/com/prezi/pride/cli/commands/ListCommand.java

```diff
@@ -68,12 +85,17 @@ public String formatModule(Module module) throws Exception {
 			VcsStatus status = module.getVcs().getSupport().getStatus(moduleDirectory);
 			String branch = status.getBranch();
 			StringBuilder line = new StringBuilder();
+
 			line.append(status.hasUnpublishedChanges() ? 'M' : ' ');
 			line.append(status.hasUncommittedChanges() ? 'M' : ' ');
 			line.append(' ').append(module.getName());
-			line.append(' ');
+
+			line.append(new String(new char[moduleMaxLength + 2 - module.getName().length()]).replace("\0", " "));
```

Comment:
:police_car: :police_car: :police_car: !

Not my invention though.. i just stackoverflow-ed how to repeat a string in java and picked the one which looked coolest :D 

I'll replace it with Strings.repeat().